Gemfile.lock

```diff
@@ -1,94 +1,94 @@
+    actioncable (6.0.3.4)
+      actionpack (= 6.0.3.4)
       nio4r (~> 2.0)
       websocket-driver (>= 0.6.1)
-    actionmailbox (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      activejob (= 6.0.3.2)
-      activerecord (= 6.0.3.2)
-      activestorage (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionmailbox (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      activejob (= 6.0.3.4)
+      activerecord (= 6.0.3.4)
+      activestorage (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
       mail (>= 2.7.1)
-    actionmailer (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      actionview (= 6.0.3.2)
-      activejob (= 6.0.3.2)
+    actionmailer (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      actionview (= 6.0.3.4)
+      activejob (= 6.0.3.4)
       mail (~> 2.5, >= 2.5.4)
       rails-dom-testing (~> 2.0)
-    actionpack (6.0.3.2)
-      actionview (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionpack (6.0.3.4)
+      actionview (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
       rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actiontext (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      activerecord (= 6.0.3.2)
-      activestorage (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actiontext (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      activerecord (= 6.0.3.4)
+      activestorage (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
       nokogiri (>= 1.8.5)
-    actionview (6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionview (6.0.3.4)
+      activesupport (= 6.0.3.4)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activejob (6.0.3.2)
-      activesupport (= 6.0.3.2)
+    activejob (6.0.3.4)
+      activesupport (= 6.0.3.4)
       globalid (>= 0.3.6)
-    activemodel (6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activerecord (6.0.3.2)
-      activemodel (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activestorage (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      activejob (= 6.0.3.2)
-      activerecord (= 6.0.3.2)
+    activemodel (6.0.3.4)
+      activesupport (= 6.0.3.4)
+    activerecord (6.0.3.4)
+      activemodel (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
+    activestorage (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      activejob (= 6.0.3.4)
+      activerecord (= 6.0.3.4)
       marcel (~> 0.3.1)
-    activesupport (6.0.3.2)
+    activesupport (6.0.3.4)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
       zeitwerk (~> 2.2, >= 2.2.2)
     aws-eventstream (1.1.0)
-    aws-partitions (1.353.0)
-    aws-sdk-core (3.104.3)
+    aws-partitions (1.399.0)
+    aws-sdk-core (3.109.3)
       aws-eventstream (~> 1, >= 1.0.2)
       aws-partitions (~> 1, >= 1.239.0)
       aws-sigv4 (~> 1.1)
       jmespath (~> 1.0)
-    aws-sdk-kms (1.36.0)
-      aws-sdk-core (~> 3, >= 3.99.0)
+    aws-sdk-kms (1.39.0)
```

Comment:
### [aws-sdk-kms](https://github.com/aws/aws-sdk-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.36.0 → 1.39.0

[_[change-log](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sdk-kms/CHANGELOG.md), [source-code](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sdk-kms)_]





<details>
<summary>Commits</summary>

A change of **62** commits. See the full changes on [the compare page](https://github.com/aws/aws-sdk-ruby/compare/v1.36.0...v1.39.0).

These are the first 10 commits:
- (6402970) [Support uri unsafe characters in proxy_uri](https://github.com/aws/aws-sdk-ruby/commit/640297066fff98d248ba957e85858b921e25f1e1)
- (c199fcd) [Replace UUIDTools with SecureRandom.uuid](https://github.com/aws/aws-sdk-ruby/commit/c199fcde1c22b9416d8748521eac4d7e1bc2df03)
- (47778ab) [Backport SecureRandom.uuid for Ruby 1.8.7](https://github.com/aws/aws-sdk-ruby/commit/47778abf5234255f4eeb73f4c5e070913739fddb)
- (262a383) [Update hosted_zone_collection.rb](https://github.com/aws/aws-sdk-ruby/commit/262a38307d84dd2f744413c4e19b7ab86c6ecd26)
- (26e34ea) [Fix a performance issue when calling CredentialProviders::De…](https://github.com/aws/aws-sdk-ruby/commit/26e34ea1c0eeb8cbaf7518c7587eae17a426d714)
- (b502dd8) [fix vault.archives account_id](https://github.com/aws/aws-sdk-ruby/commit/b502dd819693c0f0aee6df42ad214eb47768edb6)
- (ed20b13) [Small EC2 documentation fixes.](https://github.com/aws/aws-sdk-ruby/commit/ed20b13d6961e29db5ef57bd3e8912836253259a)
- (cab66b4) [Merge pull request #487 from aviddiviner/master](https://github.com/aws/aws-sdk-ruby/commit/cab66b48bddd7ba6869d917c1de2c385b8387165)
- (ad86bbd) [Revert "Each AWS::Core::Http::NetHttpHandler instance gets i…](https://github.com/aws/aws-sdk-ruby/commit/ad86bbd79a7f42201f79f3b5de382ea148c7bde8)
- (522892d) [Bump version to v1.36.1](https://github.com/aws/aws-sdk-ruby/commit/522892dab7a0b7a57dcad5fb9f461066efbb583e)


</details>